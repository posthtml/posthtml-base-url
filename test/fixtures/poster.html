<video width="250" poster="flower.jpg">
  <source src="media/flower.webm" type="video/webm">
  <source src="media/flower.mp4" type="video/mp4">
</video>
