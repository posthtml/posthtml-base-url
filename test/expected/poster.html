<video width="250" poster="https://example.com/flower.jpg">
  <source src="https://example.com/media/flower.webm" type="video/webm">
  <source src="https://example.com/media/flower.mp4" type="video/mp4">
</video>
